HHH-15160 - Adds the Postgis distance operators (#6466)
hibernate/hibernate-orm: documentation/src/main/asciidoc/userguide/chapters/query/spatial/Spatial.adoc

|================================
^(1)^ Argument Geometries need to have the same dimensionality.

Note that beyond the common spatial functions mentioned above, Hibernate may define additional spatial functions for each database dialect. These will be documented in the
Database notes below.

=== Database notes
[[spatial-configuration-dialect-postgresql]]
Postgresql::

The Postgresql dialect has support for the https://postgis.net/[Postgis spatial extension], but not the Geometric types mentioned in the
https://www.postgresql.org/docs/current/datatype-geometric.html[Postgresql documentation].

> In addition tot he common spatial functions, the following functions are supported:

Reviewer: ```suggestion
In addition to the common spatial functions, the following functions are supported:
```
